R/resample.R

```diff
@@ -75,3 +75,16 @@ dim.resample <- function(x, ...) {
 obj_sum.resample <- function(x, ...) {
   paste0("resample [", big_mark(nrow(x)), " x ", big_mark(ncol(x)), "]")
 }
+
+`$.resample` <- function(x, name) {
```

Comment:
I'm not sure that this is a good idea - if you're going to override `$` and `[[` you also need to override `length` and `names` to make them consistent. And then you'll need to override `str()` so it display the correct structure for the model.